routes/keyword.js

```diff
@@ -36,10 +39,17 @@ router.get('/:word', (req, res) => {
   const word = req.params.word
   Keyword.find({ word })
     .lean()
-    .then(words => res.send(words[0].votes))
+    .then((wordList) => {
+      if (wordList.length === 0) {
```

Comment:
fineOne will make one of the article tests fail but im not going to bother fixing it because Kirin is basically rewriting all of them in his pr